makefiles/Makefile.devhub-content

```diff
@@ -27,14 +27,13 @@ help: ## Show this help message
 
 update-snooty:
 	cd ../../snooty-devhub; \
-	git checkout master; \
-	git pull; \
+	git fetch --all; \
```

Comment:
let's remove the makefiles from this PR -- the way the merge was resolved will break our devhub-content builds.

Reply:
Ah my apologies, was so busy exporting the app from stitch that I forgot to stash these changes